marshmallow/utils.py

```diff
@@ -334,16 +334,31 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
```

Comment:
Shouldn't compat.py stick to pure Python stuff, so that it can be imported anywhere without circular import issues? For instance, what if we need to import compat.py in utils.py some day?